app/src/atoms/MenuList/DropdownMenu.tsx

```diff
@@ -26,52 +26,104 @@ export interface DropdownMenuProps {
+  title?: string
 }
 
 // TODO: (smb: 4/15/22) refactor this to use html select for accessibility
 
 export function DropdownMenu(props: DropdownMenuProps): JSX.Element {
-  const { filterOptions, onClick, currentOption } = props
+  const {
+    filterOptions,
+    onClick,
+    currentOption,
+    width = '9.125rem',
+    dropdownType = 'rounded',
+    title,
+  } = props
   const [showDropdownMenu, setShowDropdownMenu] = React.useState<boolean>(false)
-  const toggleSetShowDropdownMenu = (): void =>
+  const toggleSetShowDropdownMenu = (): void => {
     setShowDropdownMenu(!showDropdownMenu)
+  }
   const dropDownMenuWrapperRef = useOnClickOutside<HTMLDivElement>({
-    onClickOutside: () => setShowDropdownMenu(false),
+    onClickOutside: () => {
+      setShowDropdownMenu(false)
+    },
   })
 
+  const DROPDOWN_STYLE = css`
+    flex-direction: ${DIRECTION_ROW};
+    background-color: ${COLORS.white};
+    cursor: pointer;
+    padding: ${SPACING.spacing8};
+    border: 1px ${BORDERS.styleSolid}
+      ${showDropdownMenu ? COLORS.blue50 : COLORS.grey50};
+    border-radius: ${dropdownType === 'rounded'
+      ? BORDERS.radiusRoundEdge
+      : BORDERS.radiusSoftCorners};
+    align-items: ${ALIGN_CENTER};
+    justify-content: ${JUSTIFY_SPACE_BETWEEN};
+    width: ${width};
+
+    &:hover {
+      border: 1px ${BORDERS.styleSolid}
+        ${showDropdownMenu ? COLORS.blue50 : COLORS.grey55};
+    }
+
+    &:active {
+      border: 1px ${BORDERS.styleSolid} ${COLORS.blue50};
+    }
+
+    &:focus-visible {
+      border: 1px ${BORDERS.styleSolid} ${COLORS.grey55};
+      outline: 2px ${BORDERS.styleSolid} ${COLORS.blue50};
+      outline-offset: 2px;
+    }
+
+    &:disabled {
+      background-color: ${COLORS.transparent};
+      color: ${COLORS.grey40};
+    }
+  `
+
   return (
-    <>
+    <Flex flexDirection={DIRECTION_COLUMN}>
+      {title !== undefined && (
+        <StyledText as="labelSemiBold" paddingBottom={SPACING.spacing8}>
+          {title}
+        </StyledText>
+      )}
       <Flex
-        flexDirection={DIRECTION_ROW}
-        alignItems={ALIGN_CENTER}
-        justifyContent={JUSTIFY_SPACE_BETWEEN}
-        width="9.125rem"
-        onClick={toggleSetShowDropdownMenu}
-        border={BORDERS.lineBorder}
-        borderRadius={BORDERS.radiusRoundEdge}
-        padding={SPACING.spacing8}
-        backgroundColor={COLORS.white}
-        css={css`
-          cursor: pointer;
-        `}
+        onClick={(e: MouseEvent) => {
+          e.preventDefault()
+          toggleSetShowDropdownMenu()
+        }}
+        css={DROPDOWN_STYLE}
+        ref={dropDownMenuWrapperRef}
       >
         <StyledText css={TYPOGRAPHY.pSemiBold}>{currentOption.name}</StyledText>
-        <Icon
-          height={TYPOGRAPHY.lineHeight16}
-          name={showDropdownMenu ? 'chevron-up' : 'chevron-down'}
-        />
+        {showDropdownMenu ? (
+          <Icon
+            height={TYPOGRAPHY.lineHeight16}
```

Comment:
lineHeight is for text only so recommend you to use `1 rem`

```suggestion
            height='1rem'
```

Reply:
![Screenshot 2024-03-14 at 1 31 41 PM](https://github.com/Opentrons/opentrons/assets/474225/1f284f3f-d9c1-4142-8aea-f81db47f8913)

maybe using size='0.75rem' would be better.